spec/models/verse_spec.rb

```diff
@@ -58,4 +58,12 @@
       )
     end
   end
+
+  describe 'verse of the day' do
```

Comment:
This generally would be `describe '.verse_of_the_day'` because it's testing a method. Note the dot, it's a class method rather than an instance method (you'd use `#verse_of_the_day` in that case.

Reply:
Awesome!! Thanks!